packages/by-name/contrast/package.nix

```diff
@@ -41,10 +38,29 @@ let
     ];
   };
 
-  launchDigest = builtins.readFile "${microsoft.runtime-class-files}/launch-digest.hex";
-
-  runtimeHandler = lib.removeSuffix "\n" (
-    builtins.readFile "${microsoft.runtime-class-files}/runtime-handler"
+  # Reference values that we embed into the Contrast CLI for
+  # deployment generation and attestation.
+  embeddedReferenceValues = builtins.toFile "reference-values.json" (
+    builtins.toJSON {
+      aks = {
+        snp = {
+          minimumTCB = {
+            bootloaderVersion = 3;
+            teeVersion = 0;
+            snpVersion = 8;
+            microcodeVersion = 115;
+          };
+        };
+        trustedMeasurement = lib.removeSuffix "\n" (
+          builtins.readFile "${microsoft.runtime-class-files}/launch-digest.hex"
+        );
+      };
+      bareMetalTDX = {
+        trustedMeasurement = lib.removeSuffix "\n" (
+          builtins.readFile "${kata.runtime-class-files}/launch-digest.hex"
+        );
+      };
```

Comment:
This uses the launch measurement calculated for SNP. Should we maybe use a different fake value until we do the launch calculations for TDX? My fear is that using the launch measurement for SNP might lead to confusion.

Reply:
Currently this combines the hashes of SNP and TDX software in the launch digest, which, imo, should be acceptable for now, since we don't actually deploy SNP and TDX side-by-side just yet. Making them individual digests obviously needs to happen eventually, but we'll need to tweak quite some code in the node-installer to accept multiple runtime-handlers and install based on configuration here, which, I think, is out of scope for this PR.

Reply:
Ah, fair enough, I forgot that https://github.com/edgelesssys/contrast/pull/730/commits/5527f8d47054777533a51ff53c420920fce5d4fc hasn't been merged yet, my bad.